civis_jupyter_notebooks/assets/git-clone

```diff
@@ -1,8 +1,17 @@
-#!/bin/bash
+#!/usr/bin/env/python
```

Comment:
should we rename this file as well?

Reply:
`clone-repository`?

Reply:
That's fine. Or just preface with `civis`

Reply:
It is `#!/usr/bin/env python` (with a space).